js/main.js

```diff
@@ -0,0 +1,49 @@
+const checkCorrectNumber = (value) => {
+  if (Number.isNaN(value) === true || typeof value !== 'number' || Number.isFinite(value) !== true) {
+    throw new RangeError('Числа должны быть положительными или 0.');
+  }
+
+  if (value < 0) {
+    value = 0;
+  } else if (value > Number.MAX_SAFE_INTEGER) {
+    value = Number.MAX_SAFE_INTEGER;
+  }
+
+  return value;
+};
+
+const getRandomNumber = (minNum = 0, maxNum = Number.MAX_SAFE_INTEGER, accuracy = 0) => {
+  try {
+    minNum = checkCorrectNumber(minNum);
+    maxNum = checkCorrectNumber(maxNum);
+    accuracy = checkCorrectNumber(accuracy);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error.name);
+    // eslint-disable-next-line no-console
+    console.log(error.message);
+  }
```

Comment:
Try catch здесь выглядит излишне, если мы передаём неверные данные, то должны выкинуть ошибку, а не обработать её.
Не нужно переписывать параметры
```suggestion
    const minCheckedNum = checkCorrectNumber(minNum);
    const maxCheckedNum = checkCorrectNumber(maxNum);
    const checkedAccuracy = checkCorrectNumber(accuracy);
```

Reply:
Убрал Try catch, убрал переписование в параметрах, заменил minCheckedNum и maxCheckedNum на let  для использования ниже при деструктуризации